src/client/components/Sidebar/ObjectExpertise.less

```diff
@@ -1,11 +1,27 @@
 @import (reference) '../../styles/custom.less';
 
-.ObjectExpertise__more {
-  padding-top: 14px;
+.ObjectExpertise {
+  &__buttons-wrap {
+    display: flex;
+    justify-content: space-around;
+    padding-top: 14px;
 
-  & > a {
-    color: @blue-botticelli;
-    font-weight: 500;
+    &-all {
+      & > a {
```

Comment:
Select elements by class (add classname to link)

Reply:
++